org.gnome.GTG.json

```diff
@@ -0,0 +1,77 @@
+{
+  "app-id": "org.gnome.GTG",
```

Comment:
Does it make sense for the app to use org.gnome app-id? See https://discourse.gnome.org/t/official-proposal-how-we-define-gnome-software/3371

Reply:
Is that proposal moving forward? We are might have to change names too, since we are using Gnome's trademark. It'd be better to defer changing the app id until both the proposal is in effect and we figure out what to do with the name (probably for 0.5-0.6).

Reply:
I will ask around

Reply:
I believe even in the event of that proposal it wouldn't apply to an existing app (unless the app wanted to move explicitly) e.g.

> Will I have to change the ID of my app?
> 
> No. The guidelines for app IDs only apply to new apps.

So it's not a blocker to me.

Reply:
As the app is not published anywhere, it's a new app to me and should be used as an opportunity to use a proper id.

Reply:
I asked Allan, it seems to be okay to keep that app-id.